source/integration.rst

```diff
@@ -1,7 +1,20 @@
 Integration
 -----------
 
-You can follow dedicated integration guides for each cloud management framework:
+Integration of cloud stacks into EGI FedCloud follows a well-defined path, with certain steps which need to be taken, depending on the cloud stack in question. By integration here, we refer to the proper interoperation with EGI infrastructure services such as accounting, monitoring, authentication and authorisation, _etc_. These configurationsmake your site discoverable and usable by the communities you wish to support, and allow EGI to support you in operational and technical matters.
+
+Integration of these services implies specific configuration actions which you need to take on your site. These aim to be unintrusive and are mostly to facilitate access to your site by the communities you wish to support, without interfering with normal operations. This can be summarised essentially as :
+
+  1. Network configuration 
+  1. Permissions configuraiton
```

Comment:
Typi

Reply:
> On long lines: I started this by copy-pasting from the wiki, so it's full of long lines, I don't like them either but I didn't have the time/willingness to do nice wrapping. Is there any automatic way of doing that?

Only thing I could find is https://github.com/mrmlnc/vscode-remark